services/vision/mlvision/segmenter3d.go

```diff
@@ -0,0 +1,12 @@
+package mlvision
+
+import (
+	"errors"
+
+	"go.viam.com/rdk/services/mlmodel"
+	"go.viam.com/rdk/vision/segmentation"
+)
+
+func attemptToBuild3DSegmenter(mlm mlmodel.Service) (segmentation.Segmenter, error) {
```

Comment:
is this needed?

Reply:
Sorta kinda!! It's like a promise to do this later 🥺

Reply:
hah. if it's not needed right now, I'd just delete it or add a TODO with a JIRA

Reply:
Okok... TODO added with the Jira ticket number on it